.github/workflows/golangci.yml

```diff
@@ -17,10 +17,10 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: '1.20'
+          go-version: '1.21'
       - uses: actions/checkout@v3
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3.6.0
         with:
-          version: v1.53.3
-          args: --timeout 10m
+          version: v1.54.2
+          args: --timeout 5m
```

Comment:
I think we bumped this timeout to 10m at some point. Not sure exactly why other than general slow timings. Let's see how it fairs at 5m again I guess.